app/javascript/packs/vertical_nav.js

```diff
@@ -0,0 +1,18 @@
+import { VerticalNavWrapper as VerticalNav } from 'Navigation/components/VerticalNav'
+import { safeFromJsonString } from 'utilities/json-utils'
+
+import '@patternfly/react-core/dist/styles/base.css'
```

Comment:
If you have issues importing all PF4 base styles, you can try importing partials like [here](https://github.com/3scale/porta/pull/761/files#diff-bfd5a6ba0188cf52dcbccd0f3ce63c02R1)